src/GafferScene/SetVisualiser.cpp

```diff
@@ -73,16 +73,19 @@ std::vector<Override> unpackOverrides( const CompoundDataPlug *plug )
 	std::string name;
 	for( CompoundDataPlug::MemberPlugIterator it( plug ); !it.done(); ++it )
 	{
-		ConstColor3fDataPtr color = runTimeCast<Color3fData>( plug->memberDataAndName( it->get(), name ) );
-		if( !color )
+		// This will fail if the member has been disabled, or has no name
+		if( ConstDataPtr plugData =  plug->memberDataAndName( it->get(), name ) )
 		{
-			throw IECore::Exception( boost::str( boost::format(
-				"Color Override value for \"%s\" is not a Color3f") % name )
-			);
-		}
-		if( !name.empty() )
-		{
-			overrides.push_back( Override( name, color ) );
+			if( ConstColor3fDataPtr asColor = runTimeCast<const Color3fData>( plugData ) )
```

Comment:
Ownership is dealt with by `plugData`, so this could/should be a raw pointer.

Reply:
As Tom has politely pointed out by another channel, we do retain ownership of `asColor` here, because it ends up in `overrides`, which stores `Color3fPtr` rather than `Color3f` as I was assuming.